README.md

```diff
@@ -1,72 +1,71 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
-# Project Name
-
-> Description the project.
+# metrics-webapp
 
+> This metrics web-app is build as a mobile web application to check a list of metrics (numerical values). It is created using React and Redux. I have used an external API to show a covid-19 statistical data of specific continent and country. This webapp has a home page to show a filtered items of continent and a countries page to show a details data of a particular country.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML, CSS
+- Git, GitHub, Webpack
+- React, Redux
 
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
+### Prerequisites
 
+-- Any text editor such as vscode.
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
+### To launch the project locally:
 
-### Setup
+#### Step 1:
 
-### Install
+- Clone this repo locally using git in the command line and the following command
 
-### Usage
+```bash
+> git clone git@github.com:Addisu87/metrics-webapp.git
+```
 
-### Run tests
+- Alternatively, you can just download the complete zip file and extract the folder in your directory
 
-### Deployment
+#### Step 2:
 
+- In the terminal, you can run:
 
+```bash
+> cd metrics-webapp
+> npm install
+> code .
+> npm start
+```
 
-## Authors
+- The page should open in localhost:3000
+
+## Live Demo (if available)
 
-👤 **Author1**
+[Live Demo Link](https://react-metrics-webapp.netlify.app/)
```

Comment:
- [ ] The deployed version of your application doesn't display the pages. This is what I see when I click the link you provided in the `README.md` file:

![image](https://user-images.githubusercontent.com/77106962/181653211-fe12765f-e6ef-440f-9177-7bceeed8fbf9.png)

#### Kindly deploy the project properly and ensure that all pages and their features work properly in the deployed version. 👍